voila/execute.py

```diff
@@ -35,13 +35,30 @@ def strip_code_cell_errors(cell):
     return cell
 
 
+def strip_code_cell_warnings(cell):
+    """Strip any warning outputs and traceback from a code cell."""
+    # There is no 'outputs' key for markdown cells
+    if 'outputs' not in cell:
+        return cell
+
+    outputs = cell['outputs']
+
+    warning_outputs = [output for output in outputs if output['output_type'] == 'stream' and output['name'] == 'stderr']
+
+    for output in warning_outputs:
+        output['name'] = ''
```

Comment:
Shouldn't you set the "text" key to an empty string as well?

Reply:
I thought about this, but by setting the "name" key to an empty string, the warning message doesn't get printed to any output stream and I thought it won't make a difference if I'd change/emptied the text. But I'm not sure I got the logic right :grimacing:. Am I making any sense?

Reply:
Yes! Maybe the front-end does not show anything by default if the name is empty, I did not check what the logic is. But just to be sure I would set the text to empty string too...

Thinking about it, it might be even better to remove those outputs from the cell outputs list. What do you think?

Reply:
I agree it'll be better to remove the warning outputs entirely.

Reply:
Updated the PR! Thanks a lot for the feedback :)